ckanext/validation/logic/auth.py

```diff
@@ -0,0 +1,39 @@
+import ckan.plugins.toolkit as tk
+
+
+def _get_auth_functions():
```

Comment:
This shouldn't need to start with an underscore. It's not internal to this file; it's called by the plugin module.

Reply:
\+

Reply:
This will be fixed and pushed by Alex shortly.